src/pudl/transform/eia923.py

```diff
@@ -1223,3 +1223,46 @@ def _core_eia923__fuel_receipts_costs(
     frc_df = frc_df.dropna(subset=["energy_source_code"])
 
     return frc_df
+
+
+@asset
+def _core_eia923__cooling_system_information(
+    raw_eia923__cooling_system_information: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transforms the eia923__cooling_system_information dataframe."""
+    csi_df = raw_eia923__cooling_system_information
+    # Generic cleaning
+    csi_df = csi_df.pipe(pudl.helpers.fix_eia_na).pipe(pudl.helpers.convert_to_date)
+
+    # Convert cooling_status from code to description
+    csi_df["cooling_status"] = csi_df.cooling_status.str.upper().map(
+        pudl.helpers.label_map(
+            CODE_METADATA["core_eia__codes_operational_status"]["df"],
+            from_col="code",
+            to_col="description",
+            null_value=pd.NA,
+        )
+    )
+
+    # Combine the annual rates with the monthly rates
+    cols = {
+        "withdrawal_rate_gallons_per_minute": "annual_withdrawal_rate_tenth_cubic_feet_per_second",
+        "discharge_rate_gallons_per_minute": "annual_discharge_rate_tenth_cubic_feet_per_second",
+        "consumption_rate_gallons_per_minute": "annual_consumption_rate_tenth_cubic_feet_per_second",
+    }
+    for new_col, old_col in cols.items():
+        csi_df.loc[csi_df["report_date"].dt.year.isin([2008, 2009]), new_col] = csi_df[
+            old_col
+        ]
+
+    # In 2008 and 2009 the rate columns are clearly labeled as 0.1 cubic feet / second
+    # but that rate persists until 2013 when they change it to gallons/min.
+    # Convert tenth cubic feet/second columns to gallons/minute.
+    conversion = 10 / 60 * 7.48052
+
+    for new_col in cols:
+        csi_df.loc[csi_df["report_date"].dt.year.isin(range(2008, 2013)), new_col] = (
+            csi_df[new_col] * conversion
+        )
```

Comment:
Something weird happens between 2012 and 2013. The values for the rates go way up! The column also changes from 0.1 cubic feet / second to gallons per minutes, but even with the conversion in place the values spike in 2013. Here is the average rate for each year: 

<img width="300" alt="Screen Shot 2024-02-16 at 12 36 01 PM" src="https://github.com/catalyst-cooperative/pudl/assets/49878195/790a7b90-b640-4914-ba1b-7e756c23e462">

<img width="310" alt="Screen Shot 2024-02-16 at 12 35 36 PM" src="https://github.com/catalyst-cooperative/pudl/assets/49878195/652fc880-1ec6-4d2c-91ec-aacea4534051">

<img width="325" alt="Screen Shot 2024-02-16 at 12 35 21 PM" src="https://github.com/catalyst-cooperative/pudl/assets/49878195/a5ee4eb6-15c1-41af-bbbb-618f2abd4b16">

Reply:
I don't think this has to do with the annual vs monthly because that only applies for 2008 and 2009 and this issue goes through 2012. Also these values are the annual average.